react-json-tree/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for react-json-tree v0.6.5
+// Project: https://github.com/alexkuz/react-json-tree/
+// Definitions by: Grant Nestor <https://github.com/gnestor/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "react-json-tree" {
```

Comment:
could you strip `declare module`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred